vcluster/deploy/upgrade/distro-migration.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Migrate from k3s to Kubernetes
+sidebar_label: k3s to k8s Migration
+sidebar_position: 2.2
+description: Learn how to migrate your virtual clusters from k3s distro to Kubernetes (k8s) distro.
+---
+
+import BasePrerequisites from '../../../docs/_partials/base-prerequisites.mdx';
+
+Starting with vCluster 0.25.0, you can migrate an existing virtual cluster from k3s distro to Kubernetes (k8s) distro. This feature allows you to benefit from the full Kubernetes implementation while preserving your existing workloads and configuration.
+
+## Prerequisites {#prerequisites}
+
+<BasePrerequisites />
+
+Additionally, you need:
+- `vCluster` CLI version 0.25.0 or higher
+- An existing virtual cluster using k3s distro
+
+## Migration process {#migration-process}
+
+The migration happens automatically when you change the distro from k3s to k8s during an upgrade. There are two supported methods for initiating the migration:
+
+1. Using the CLI with direct flags (`--distro=k8s`)
```

Comment:
This wasn't tested afaik, just using config. You could try it pretty easily.

Reply:
Good catch, this command doesn't actually work, only `values.yaml` is supported.